staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5489,7 +5489,7 @@ type ReplicationControllerSpec struct {
 	// insufficient replicas are detected. This takes precedence over a TemplateRef.
 	// The only allowed template.spec.restartPolicy value is "Always".
 	// More info: https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller#pod-template
-	// +optional
+	// +k8s:required
 	Template *PodTemplateSpec `json:"template,omitempty" protobuf:"bytes,3,opt,name=template"`
```

Comment:
for +required fields, typically they are not omitempty either.  I want to confirm if this is actually something that can be changed (I don't believe it is)

Reply:
Latest guidance is that required fields should also have omitempty, so please leave as is